com.github.sdbg.debug.ui/META-INF/MANIFEST.MF

```diff
@@ -21,3 +21,5 @@ Bundle-ActivationPolicy: lazy
 Bundle-RequiredExecutionEnvironment: JavaSE-1.6
 Bundle-ClassPath: .
 Bundle-Vendor: %providerName
+Export-Package: com.github.sdbg.ui.launchers
```

Comment:
Let's change this package to com.github.sdbg.debug.ui as suggested.
("sdbg" and "debug" in one package sounds like a tautology, but we already have all other packages that way...)